LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -49,7 +49,11 @@
  Any help would be greatly appreciated.
  */
 
+#if defined(LTH_APP_EXTENSIONS)
```

Comment:
Hmm, the only place for the `#define` to be placed in for either `#ifdef` or `#if defined()` to work is in the `.pch` file; otherwise in the project. I just wanted to know all cases, so I can update the README.

`#ifdef` and `#if defined(X)` are basically the same thing, just that with the second approach you can add more clauses:`#if defined(X) && VERSION > 3`, while the first just tests if the macro/value exists.